src/tir/schedule/primitive/cache_read_write.cc

```diff
@@ -297,7 +297,12 @@ bool CalculateAffineFlag(const ScheduleState& self, const StmtSRef& block_sref)
  * \param stage The stage to be inserted
  * \return A SeqStmt, the result after insertion
  */
-SeqStmt InsertCacheStage(const Stmt& stmt, int pos, const Stmt& stage) {
+Stmt InsertCacheStage(const Stmt& stmt, int pos, const Stmt& stage) {
```

Comment:
Is this change related?

Reply:
It is not related but I found this bug while testing this branch. It happens on CPU when `link-params = True` is used and `cache_write` is applied during tuning.

Reply:
Added a test case for this fix. For some reason I cannot reproduce this bug without anchor tuning. So I'm including it here.